__tests__/ReactApp.test.js

```diff
@@ -34,11 +41,26 @@ afterEach(() => {
   cleanup();
```

Comment:
We have this all over, so I'm okay with keeping it as is for consistency, but wanted to share that it's usually not necessary: 
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-cleanup